src/components/business-lookup/BusinessLookup.vue

```diff
@@ -2,55 +2,70 @@
   <div id="business-lookup">
     <div v-if="state !== States.SUMMARY">
       <v-autocomplete
-        filled no-filter append-icon="" return-object
+        v-model="selectedBusiness"
+        v-model:search-input="searchField"
```

Comment:
I need to test this -- eslint changed the code to this and then complains about it:

![image](https://github.com/bcgov/bcrs-shared-components/assets/10002889/631f64c6-b377-4e40-b6f8-e9097bf5e6ad)

Reply:
Seems to work just like the old one.